src/ColumnDisplayFormatDialog.cpp

```diff
@@ -37,7 +37,6 @@ ColumnDisplayFormatDialog::ColumnDisplayFormatDialog(DBBrowserDB& db, const sqlb
     ui->comboDisplayFormat->addItem(tr("Custom"), "custom");
 
     ui->labelDisplayFormat->setText(ui->labelDisplayFormat->text().arg(column_name));
-
```

Comment:
You should undo this change, so the PR has a clean history when squashed and merged.